lib/util/apis.dart

```diff
@@ -2,8 +2,10 @@ import 'dart:developer' as developer;
 import 'package:http/http.dart' as http;
 import 'dart:convert';
 
-Future<Map<String, dynamic>> postCommunity(String communityName, String token) async {
-  const url = 'https://qgzp9bo610.execute-api.us-east-1.amazonaws.com/prod/community';
+Future<Map<String, dynamic>> postCommunity(
+    String communityName, String token) async {
+  const url =
+      'https://qgzp9bo610.execute-api.us-east-1.amazonaws.com/prod/community';
```

Comment:
Can you keep these on the same line the way they were?

Reply:
I copied the entire api file from development, then added the api call for deletePost and saved it as a new commit. This should resolve the issues with the lines in the file.